### PR TITLE
project initial files added. Linters configs added

### PR DESCRIPTION
**Pull request by tigamadou**
- Linters added
- Logic added
- Linters errors fixed

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require('bulma')
```
Comment (lucasmazo23) [inline src/index.js]: [REQUIRED] Remember to use ES6 syntax, this line can be changed for:
```
import 'bulma'
```

image

[source](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)

**Review by lucasmazo23 — CHANGES_REQUESTED** [review]
## Changes required 🔧
### You are doing a great job so far, and the project is looking good.

### For the project to get approved kindly implement these changes:

### Required changes:
- [x] [REQUIRED] The app needs a way to toggle from C° to F°, right now it defaults to C° and doesn't have the ability to switch.
image
[source](https://www.theodinproject.com/courses/javascript/lessons/weather-app)
- [x] [REQUIRED] There's an import using commonjs, when it should be using ES6 (check my comment below)
image
[source](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)

### Happy coding! 💻
## Lucas Mazo

**Comment by tigamadou** [comment]
@lucasmazo23 Thank you very much for the feedback. All the changes have been implemented now. 👍

**Review by rOluochKe — APPROVED** [review]
Hello @tigamadou 
### You got it right, well done :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Comment by tigamadou** [comment]
@rOluochKe Thank you very much. 👍

**Pull request merged**